src/utils/number.ts

```diff
@@ -0,0 +1,4 @@
+export const addLeadingZero = (relativeTime: number): string =>
```

Comment:
Maybe let's rename`relativeTime` -> `num`?

Reply:
Fixed in f6394e444dca1bc487e2bd90f4723f7d51216760